data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset

```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
   }
 }
 
+--Generate all moon labels
+local moon_labels = {}
+
+for i, moon in ipairs(anankeGroup) do
+  moon_labels[i] = {
+    Identifier = moon.Identifier .. "Label",
+    Parent = moon.Identifier,
+    Renderable = {
+      Enabled = false,
+      Type = "RenderableLabel",
+      Text = moon.Identifier,
```

Comment:
I think this should be `moon.GUI.Name` instead

Reply:
See same comment as for: [data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset](https://github.com/OpenSpace/OpenSpace/pull/2453/files/508a1a9e1bdde4e53e9fd9d927dbd7990bdea7cc#diff-59c1a7d7c5055671ede6b9b0f84e990f8ed5bb46bf54241ded3b73e3f9f43de8)